packages/test-utils/src/ConditionalTest.ts

```diff
@@ -46,3 +46,9 @@ export function onNodeVersions(
     });
   }
 }
+
+export function skipTestOnJasmine(testBody: () => void): void {
```

Comment:
why is a new helper needed? shouldn't `skipSuiteOnJasmine` be enough? That one also supports snapshots properly

Reply:
`skipSuiteOnJasmine` will skip all test of file